src/d3a/models/area/__init__.py

```diff
@@ -388,13 +390,14 @@ def _consume_commands_from_aggregator(self):
     def tick(self):
         self._consume_commands_from_aggregator()
 
-        if ConstSettings.IAASettings.MARKET_TYPE == 2 or \
-                ConstSettings.IAASettings.MARKET_TYPE == 3:
+        if ConstSettings.IAASettings.MARKET_TYPE == 2:
```

Comment:
Nit: this was not implemented in your PR, but I think it might also be a candidate to use enums.

Reply:
Since it caught my attention, I would not do that, because IIRC these variables can be sent from d3a-web, therefore they will be JSON-serialized. Therefore in order to avoid creating custom JSON encoders/decoders just for enums I would say that integers are fine specifically for this case.

Reply:
My suggestion would have been to replace the simple `== 2` with something like ` == MarketTypeEnum.DOUBLE_SIDED_PAY_AS_BID.value`. This would make it a bit clearer to understand what `2` represents. This enum could be defined in d3a-interface so that it is shared by both d3a and d3a-web.
Am I missing something?

Reply:
In that case I agree with you, it will be much better that way! I only disagreed with changing the type of this variable from int to enum, but a separate enum that will compare its value (like the one you described) would be good to have here!

Reply:
Yes sorry, my original comment was probably too generic!